azimuth_caas_operator/utils/ansible_runner.py

```diff
@@ -80,14 +81,48 @@ async def ensure_deploy_key_secret(client, cluster: cluster_crd.Cluster):
     return base64.b64decode(secret.data["id_ed25519.pub"]).decode()
 
 
+async def get_global_extravars(client):
+    """
+    Retrieves the global extra vars from the specified secret.
+    """
+    # The secret is specified in the form namespace/name
+    secret_info = os.environ.get("GLOBAL_EXTRAVARS_SECRET")
+    if not secret_info:
+        return {}
```

Comment:
I think the functional test is currently testing this path, could you update that so it is exercising our "usual" path of injecting something please? It can be total junk we put in there, as long as the yaml parses OK.